_posts/2025-02-13-cockpit-333.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 333
+author:
+date: '2025-02-13'
+tags: podman, files
+slug: cockpit-333
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 333 🥃, cockpit-podman 101, and cockpit-files 16:
+
+
+## Podman: Automatically start podman.socket
```

Comment:
This needs to go. This was already released in version 100 in https://cockpit-project.org/blog/cockpit-332.html , someone just forgot to drop the release-note label :cry:

Reply:
Oops.  Dropped.